common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -93,6 +94,11 @@ public MinimapOverlay() {
                 new OverlaySize(DEFAULT_SIZE, DEFAULT_SIZE));
     }
 
+    public void scale(double scaler) {
```

Comment:
```suggestion
    public void scale(double modifier) {
```

Reply:
Also technically you should call scale.touched() so this is persisted

Reply:
Is it not like storage where it automatically calls that?! I think I noticed the changes not persisting so imma change that

Reply:
it definitely should automatically call touched or else sometimes they get saved (when you change the config) and other times they do not (saying this because most of the Config#setValue caller do not)

Reply:
it will be fixed when it's refactored